docs/flink-queries.md

```diff
@@ -436,7 +436,7 @@ SELECT * FROM prod.db.table$partitions;
 | {20211002, 10} | 1            | 1          | 0       |
 
 Note:
-For unpartitioned tables, the partitions table will contain only the record_count and file_count columns.
+For unpartitioned tables, the partitions table will contain only the record_count, file_count and total_data_size_in_bytes columns.
```

Comment:
This statement needs to be updated. Unpartitioned table will have few more fields and previous PR missed to update this. 

https://github.com/apache/iceberg/blob/4e6c7bad35ae3886378ec50a79dac0368df875a8/core/src/main/java/org/apache/iceberg/PartitionsTable.java#L96-L105

Reply:
@ajantha-bhat Good catch, fixed.